phpBB/includes/functions_admin.php

```diff
@@ -2573,6 +2573,11 @@ function prune($forum_id, $prune_mode, $prune_date, $prune_flags = 0, $auto_sync
 		$sql_and .= ' AND topic_status = ' . ITEM_MOVED . " AND topic_last_post_time < $prune_date";
 	}
 
+	if ($prune_limit > 0)
+	{
+		$sql_and .= " LIMIT $prune_limit";
```

Comment:
I wished to avoid unnecessarily adding LIMIT if its not needed.  The current implementation for `$db->sql_query_limit()` for MySQL always adds LIMIT, even if LIMIT is 0.  If LIMIT is passed as 0 it changes $total to 18446744073709551615.

As far as adding $prune_limit to the event, I do believe it should be done.  I just am unclear on all the procedures in phpBB for altering events.